var/spack/repos/builtin/packages/vectorclass-version2/package.py

```diff
@@ -19,4 +19,4 @@ class VectorclassVersion2(Package):
     version('2.01.04', sha256='7885c343b1af9eb940f4debdd7cd19544130a06ed70e0000e1a8471fb9c15118')
 
     def install(self, spec, prefix):
-        install_tree('.', prefix)
+        install_tree('.', prefix.include)
```

Comment:
Can you add a code comment above explaining why we do that? (I assume it just ships C++ templates, but it would be good to explain the rationale for the next person modifying the package)